src/fe-common/irc/fe-whois.c

```diff
@@ -406,6 +416,7 @@ static void event_whois_default(IRC_SERVER_REC *server, const char *data)
 void fe_whois_init(void)
 {
 	settings_add_bool("lookandfeel", "whois_hide_safe_channel_id", TRUE);
+	settings_add_bool("lookandfeel", "print_whois_rpl_in_active_window", FALSE);
```

Comment:
I'm not a fan of the setting name.  Perhaps `whois_reply_active_window`

```suggestion
	settings_add_bool("lookandfeel", "whois_reply_active_window", FALSE);
```